unfetter-discover/unfetter-ui
# Issue 956 (Navigation in the baseline wizard)

Fixes unfetter-discover/unfetter#956

Ability to navigate forward back in the wizard through capability group and capability selection.  There are some anomalies that will be rectified once the category property is added to the capability object in the unfetter-discover/stix repo.

There are also some CSS anomalies with the baseline wizard sidebar - these will be resolved in a subsequent PR as well.  This includes current wizard step highlighting, which needs to be added as well.

## To test:
1. Checkout this branch
2. In Baselines, start a new one
3. As you select capability groups, they appear as new wizard steps in the sidebar.
4. Click `Continue` to proceed to capability selection.  Each selected capability will be added to the sidebar under its respective capability group, but capabilities under each capability group in the sidebar will be the same for now (see note above).
5. Clicking Back and Continue should track the current group/capability.
